lib/commands/scan.ts

```diff
@@ -19,7 +19,7 @@ import { flags } from '@oclif/command';
 import type { LocalBalenaOsDevice } from 'balena-sync';
 import Command from '../command';
 import * as cf from '../utils/common-flags';
-import { getVisuals, stripIndent } from '../utils/lazy';
+import { getCliUx, stripIndent } from '../utils/lazy';
```

Comment:
The main reason for swapping `resin-cli-visuals` with `cli-ux` in this PR is that the latter allows sending spinner output to stderr instead of stdout (and does it by default), which thus avoids mixing spinner output with `--json` output which is sent to stdout. This is sufficient to allow piping stdout to `jq`, as requested in #2098. Another option would have been to disable the spinner altogether when `--json` is used. I understand that `cli-ux` is also clever enough to auto detect whether the terminal is a TTY, and prevent animation if it is not (I didn't test this, though).